src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -6674,6 +6676,15 @@ namespace HeatBalanceManager {
             state.dataSurface->FrameDivider(FrameDividerNum).DividerSolAbsorp = FrameDividerProps(16);
             state.dataSurface->FrameDivider(FrameDividerNum).DividerVisAbsorp = FrameDividerProps(17);
             state.dataSurface->FrameDivider(FrameDividerNum).DividerEmis = FrameDividerProps(18);
+            state.dataSurface->FrameDivider(FrameDividerNum).NfrcProductType = DataSurfaces::NfrcProductOptions::CurtainWall;
+
+            // look up the NFRC Product Type for Assembly Calculations using the DataSurfaces::NfrcProductName
+            for (unsigned int i = 0; i < DataSurfaces::NfrcProductName.size(); ++i) {
+                if (UtilityRoutines::SameString(DataSurfaces::NfrcProductName[i], FrameDividerAlphas(3))) {
```

Comment:
As noted above, with `NfrcProductNamesUC`, `SameString` isn't necessary. Also, this loop can `break` when a match is found. So, you might want to reorder the enum to put the default value first?